data/quarg/quarg outfits.txt

```diff
@@ -42,6 +42,8 @@ outfit "Quarg Skylance"
 	"weapon capacity" -60
 	"turret mounts" -1
 	"required crew" 1
+	"active cooling" 10
+	"cooling energy" 1
```

Comment:
I'm not really in favor of buffing the Skylance, if you want to give active cooling to something, I think it would be better on a different outfit. 
Maybe their anti-missile if you want to keep it on a weapon outfit? Since that's rather notoriously underperforming.